code/datums/weather/weather_types/snow_storm.dm

```diff
@@ -27,3 +27,25 @@
 /datum/weather/snow_storm/weather_act(mob/living/L)
 	L.adjust_bodytemperature(-rand(5,15))
 
+
+// since snowstorm is on a station z level, add extra checks to not annoy everyone
+/datum/weather/snow_storm/can_get_alert(mob/player)
+	var/standard_check = ..()
+	// dont bother checking if theyre not on the station or wouldnt be able to see the alert anyways
+	if(!standard_check || !is_station_level(player.z))
```

Comment:
This means that if they're not on a station level, they will always see the alert (at least in the context of this proc). That doesn't seem intentional with the comment. You need to split the station level check out into its own condition and return false.

Reply:
no if theyre not on a station z level it will return what it wouldve already

Reply:
Why though

Reply:
What if we want to use this weather on a non-station z-level

Reply:
tthen if its not on a station z level it will alert every mob?

Reply:
Ok I see how this works